sapp/ui/filters.py

```diff
@@ -78,8 +78,17 @@ def all_filters(session: Session) -> List[Filter]:
 
 
 def save_filter(session: Session, filter: Filter) -> None:
-    LOG.debug(f"Storing {filter}")
-    session.add(FilterRecord.from_filter(filter))
+
+    existing = session.query(FilterRecord).filter(FilterRecord.name == filter.name).first()
+    
+    if not existing:
+        session.add(FilterRecord.from_filter(filter))
+        LOG.debug(f"Adding {filter}")
+    else:
+        existing.description = filter.description
+        existing.json = filter.json
```

Comment:
Could you add `#pyre-ignore` to both of these? The type checker claims that the left hand side is `Optional[str]` but the right hand side is `graphene.String`. I'm not sure why but it seems like `graphene.String` doesn't extend `str`?

```python
# pyre-ignore [8]: `graphene.String` is not compatible with `str`
existing.description = filter.description
# pyre-ignore [8]: `graphene.String` is not compatible with `str`
existing.json = filter.json
```